webapp/src/components/CredentialsRecovery/CredentialsRecovery.js

```diff
@@ -125,12 +127,14 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
         onClick={handleOpen}
       >
         <LockIcon className={classes.iconOption} />
-        <Typography
-          variant="body1"
-          className={clsx(classes.labelOption, overrideLabelClass)}
-        >
-          {t('credentialsRecovery.credentialsRecovery')}
-        </Typography>
+        <Link>
+          <Typography
+            variant="body1"
+            className={clsx(classes.labelOption, overrideLabelClass)}
+          >
+            {t('credentialsRecovery.credentialsRecovery')}
+          </Typography>
+        </Link>
```

Comment:
@dloynaz check this

![image](https://user-images.githubusercontent.com/19317138/97912958-3a9ec580-1d13-11eb-9c27-291d1b793914.png)

Reply:
Hm. I will check it. Thanks!